further_link/endpoint/upload.py

```diff
@@ -18,8 +26,17 @@ async def upload(request):
         if not directory_is_valid(directory):
             raise web.HTTPBadRequest()
 
+        # Upload to further directory
         await do_upload(directory, work_dir, user)
 
+        # Upload to projects directory to run using miniscreen app
+        if is_miniscreen_project(directory):
+            if user is None:
+                user = default_user()
+            user_home = path.expanduser(f"~{user}")
+            project_work_dir = f"{user_home}/Desktop/Projects"
+            await do_upload(directory, project_work_dir, user, use_cache=False)
```

Comment:
the ticket was somewhat unclear about the caching/symlinking. I imagined that this would be implemented by copying the directory written above to the new location, and so when I said no cache symlink I just meant that it would be better to have an actual copy of the files there not symlinks. Now that I think about it more I can't think what would be wrong with having symlinks anyway, although I also don't see an advantage (well, maybe that the copy would be faster)

Anyway, it would be good to avoid redownloading the cacheable files each time. If you let use_cache be true it will do that by creating a second cache of the cacheable files in the project_work_dir. I think actually what we should do is not use do_upload at all here, and actually just copy the directory created by the first do_upload to the new location.